pkgs/tools/misc/pre-commit/default.nix

```diff
@@ -166,6 +201,19 @@ buildPythonApplication rec {
     # Expects `git commit` to fail when `pre-commit` is not in the `$PATH`,
     # but we use an absolute path so it's not an issue.
     "test_environment_not_sourced"
+
+    # todo: explanations for these
```

Comment:
some of these failures make sense.  I've added two hooks to the repo and they're assuming that I haven't.  I am not yet sure that they can *all* be safely ignored.